tests/sentry/integrations/aws_lambda/test_integration.py

```diff
@@ -107,7 +115,8 @@ def test_lambda_list(self, mock_react_view, mock_gen_aws_client, mock_get_suppor
         aws_external_id = "12-323"
         self.pipeline.state.step_index = 2
         self.pipeline.state.data = {
-            "arn": arn,
+            "region": region,
```

Comment:
@scefali should we make this data structure a fixture? It looks likes its being used in a couple of tests.